### ISSUE TITLE
Allow to setup remote station administrator login

### ISSUE THREAD

**Issue by yarik2720** [AzuraCast/AzuraCast#2525]
**Using Docker installation method**
Yes

**Host Operating System**
Ubuntu 16.04

**Describe the bug**
If on remote station login administrator name for icecast2 were changed - Azuracast can't get statistic, if you are changing back login to defaults as `admin` it connects well

**To Reproduce**
1. Edit on remote icecast2 server icecast.xml config file
2. Change `<admin-user>admin</admin-user>` to any other

**Expected behavior**
We need to be able to change on Azuracast settings admin login name

**Relevant Logs**
```
AzuraCast.ERROR: NowPlaying adapter error: Request returned status 401: You need to authenticate 
```

**Comment by SlvrEagle23** [comment]
@yarik2720 This is really more of a new feature request...right now, the system can only pull advanced statistics if the username for administrators is `admin`.

**Comment by yarik2720** [comment]
@SlvrEagle23 yeah, I met that problem right after update, because admin name were already changed a lot time ago.

**Comment by marekdziem** [comment]
have run into the same issue with remote username for admin not default 'admin'
